src/pymmcore_widgets/_shutter_widget.py

```diff
@@ -244,25 +238,21 @@ def _refresh_shutter_widget(self) -> None:
             if self.autoshutter:
                 self.autoshutter_checkbox.setEnabled(False)
         else:
-            self._close_shutter(self.shutter_device)
             if self.autoshutter:
                 self.autoshutter_checkbox.setEnabled(True)
-                self.autoshutter_checkbox.setChecked(True)
-                self.shutter_button.setEnabled(False)
+                self.autoshutter_checkbox.setChecked(self._mmc.getAutoShutter())
             else:
-                self.shutter_button.setEnabled(not self._mmc.getAutoShutter())
+                self.autoshutter_checkbox.setChecked(False)
+            self.shutter_button.setEnabled(True)
+            if self._mmc.getShutterOpen(self.shutter_device):
+                self._set_shutter_wdg_to_opened()
+            else:
+                self._set_shutter_wdg_to_closed()
 
             # bool to define if the shutter_device is a Micro-Manager 'Multi Shutter'
             props = self._mmc.getDevicePropertyNames(self.shutter_device)
             self._is_multiShutter = bool([x for x in props if "Physical Shutter" in x])
 
-    def _on_seq_started(self) -> None:
-        if self._mmc.getShutterOpen(self.shutter_device):
-            self._set_shutter_wdg_to_opened()
-
-    def _on_seq_stopped(self) -> None:
-        self._close_shutter(self.shutter_device)
-
```

Comment:
I agree that it's really nice to be able to have a visual indication of the state of a continuous acquisition. I just don't thnk t that the shutter way to represent that because depending on config file/autoshutter state the shutter state may or may not be tied to the sequence acquisition.  And then it's very problematic for this widget to be potentially modifying the state to be not what the user wanted.


So I think that something like you proposed:

> .maybe a good thing to do can be to modify the _on_seq_stopped method so that it won't close the shutter but only update its state (icon, etc..)

Could work really well - because unfortunately not all devices send the correct signals from the device adapter, so this is a good time to check if it's closed.

Reply:
> And then it's very problematic for this widget to be potentially modifying the state to be not what the user wanted.

This makes sense,  I haven't really been following yet here, but a general thought: widgets should be views onto a model (core), and controllers that accept user input. But nothing more.  They should modify any state that doesn't directly come from user interaction... and should otherwise attempt to show (view) the state of the core

Is that happening here? Or is it an interesting/special case?

Reply:
> but a general thought: widgets should be views onto a model (core), and controllers that accept user input. But nothing more.

100% agree! I think a nice test to add would be to:

1. Grab the core state
2. create every widget
3. check that the state is unmodified


> Is that happening here? Or is it an interesting/special case?

The goal of this PR is to move to view on the core approach. Previously there were several ways in which this widget would close shutters when it ought not to. I think that this was an approach to ensure that they stayed in sync with the core state as unfortunately the signalling from core that the shutter opened or closed is not perfect for many devices. 

I think the special cases being discussed are more so extra times that we should query the true state of the shutter - e.g. at the end of an acquisition is a good time  to check if the shutter has closed.